src/clients/paymentsRefunds/get/types.ts

```diff
@@ -0,0 +1,12 @@
+import type { MercadoPagoConfig } from '@src/mercadoPagoConfig';
+
+export declare type GetRefund = {
+  payment_id: string
+  refund_id: string;
+};
+
+export declare type GetRefundByRefundId = {
```

Comment:
por estar dentro de paymentsRefunds não poderia ser apenas Get?
```suggestion
export declare type Get = {
```

Reply:
Perfeito! Corrigido!